dist/palindrom-dom.js

```diff
@@ -2306,6 +2258,44 @@ var Palindrom = (function() {
     }
     makeInitialConnection(this);
   }
+  /**
+   * Iterates a JSON-Patch, traversing every patch value looking for out-of-range numbers
+   * @param {JSONPatch} patch patch to check
+   * @param {Function} errorHandler the error handler callback
+   * @param {*} startFrom the index where iteration starts
+   */
+  Palindrom.prototype.validateNumericsRangesInPatch = function(
+    patch,
+    errorHandler,
+    startFrom = this.palindrom.OTPatchIndexOffset
```

Comment:
Is this default value ever used? It looks incorrect (`this.palindrom` -> `this`)

Please apply the YAGNI principle and either always require this parameter or just always assume using `this.OTPatchIndexOffset`